docs/source/reference/services.md

```diff
@@ -146,8 +146,8 @@ See the GitHub repo for additional information about the [jupyterhub_idle_culler
 
 ## Externally-Managed Services
 
-You may prefer to use your own service management tools, such as Docker or
-systemd, to manage a JupyterHub Service. These **Externally-Managed
+You may prefer to use your service management tools, such as `Docker` or
+`systemd`, to manage a JupyterHub Service. These **Externally-Managed
```

Comment:
I would not put these names in code-highlighting. They are names and should be in plain text.